addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11003,13 +11012,12 @@ msgctxt "#20448"
 msgid "Could not download information"
 msgstr ""
 
+#: xbmc/video/VideoInfoScanner.cpp
 msgctxt "#20449"
-msgid "Unable to connect to remote server"
+msgid "Unable to connect to remote server. Would you like to continue scanning?"
```

Comment:
I'd still like to get rid of concatenation within Kodi itself, as it does not take RTL languages into account.